op-devstack/presets/jovian.go

```diff
@@ -0,0 +1,21 @@
+package presets
+
+import (
+	"github.com/ethereum-optimism/optimism/op-chain-ops/devkeys"
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/stack"
+	"github.com/ethereum-optimism/optimism/op-devstack/sysgo"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/intentbuilder"
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+)
+
+// WithJovianAtGenesis configures all L2s to activate the Jovian fork at genesis in sysgo mode.
+func WithJovianAtGenesis() stack.CommonOption {
```

Comment:
was this function one of the things missing to properly make the acceptance-tests (including in-memory) working?